README.md

```diff
@@ -4,10 +4,10 @@ The CUI Shortcut project demonstrates how to .
 
 ## Project Details
 
-**Built Using:** Aras 11.0 SP9
-**Browsers Tested:** Internet Explorer 11, Firefox 45 ESR, Chrome
+**Built Using:** Aras 11.0 SP12
+**Browsers Tested:** Internet Explorer 11, Firefox 62, Chrome 69
```

Comment:
The latest version of Firefox that 11.0 SP12 was certified for was FF52 as listed in the [Platform Specification](http://www.aras.com/support/documentation/11.0%20SP12/Installation%20and%20Configuration/Aras%20Innovator%2011.0%20-%20Platform%20Specifications.pdf). This just needs to be updated to say FF52 or FF45 was tested and it'll be all set to merge.